src/databricks/labs/remorph/reconcile/execute.py

```diff
@@ -65,7 +72,7 @@ def recon(
 
     recon_id = str(uuid4())
     # initialise the Reconciliation
-    reconciler = Reconciliation(source, target, database_config, report_type, schema_comparator, source_dialect)
+    reconciler = Reconciliation(source, target, database_config, report_type.lower(), schema_comparator, source_dialect)
```

Comment:
this change cascaded everywhere down.
```suggestion
    report_type = report_type.lower()
    reconciler = Reconciliation(source, target, database_config, report_type, schema_comparator, source_dialect)
```

Reply:
Done